controllers/storage.go

```diff
@@ -78,6 +79,9 @@ var ConnectAndQueryObjStore = func(ctx context.Context, log logr.Logger, endpoin
 		return false
 	}
 
+	ctx, cancel := context.WithTimeout(ctx, time.Second)
```

Comment:
similar to above (maybe less so since this is s3), is a 1 second timeout too aggressive and/or should be configurable?

Reply:
updated